book/json/README.md

```diff
@@ -76,9 +76,10 @@ checking]{.idx}[unit tests]{.idx}
 ##### Installing the Yojson Library
 
 There are several JSON libraries available for OCaml. For this chapter, we've
-picked the [Yojson](https://github.com/mjambon/yojson) library by Martin Jambon.
-It's easiest to install via OPAM by running `opam install yojson`. See
-[the installation instructions](install.html) if you haven't already got opam.
+picked the [Yojson](https://github.com/ocaml-community/yojson) library by Martin Jambon.
```

Comment:
Maybe we should fix this in a separate PR, but this feels like too much time spent on installing. We opam install things in lots of places. I would just say tell people to install yojson by typing opam install yojson. Like, in "lists and patterns", we write:

> Here, we’ll benchmark these functions using the core_bench library, which can be installed by running opam install core_bench from the command line.

which seems...fine?